Description: vo_vdpau: Explicitly include header vdpau_x11.h
 The include from inside libavcodec/vdpau.h was removed in FFmpeg
 commit d40e181bec22014a9ea312ab6837f7f0bc4f9e42
 .
 This fixes the FTBFS with ffmpeg 3.4
Author: Alexander Strasser <eclipse7@gmx.net>
Origin: upstream, commit:37932
Bug-Debian: https://bugs.debian.org/834135
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/

--- a/libvo/vo_vdpau.c
+++ b/libvo/vo_vdpau.c
@@ -34,6 +34,7 @@
 
 #include <stdio.h>
 #include <strings.h>
+#include <vdpau/vdpau_x11.h>
 
 #include "config.h"
 #include "sub/ass_mp.h"
